lib/assessment.rb

```diff
@@ -20,21 +20,21 @@ def sum(enum)
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    odd?
   end
 
   # Question 4
   # Write ruby code that returns true if every entry in the
   # numbers enumerable are even
   def every_even(numbers)
-    # Your code goes here
+    even?
   end
 
   # Question 5
   # Write ruby code that returns an array with all of the
   # entries from the words enumerable capitalized.
   def transform(words)
-    # Your code goes here
+   words.length.upcase
```

Comment:
No, this isn't right. `words.length` will give us a number, the length of `words`, and trying to capitalize a number doesn't make sense.
 
What we want to do in this question is iterate through `words` and apply a transformation (specifically, capitalization) to each word. Can you think of what tools we might need in order to be able to do that?